build.gradle

```diff
@@ -76,12 +76,12 @@ configure(coreProjects) {
         mavenCentral()
 
         // Uncomment this to test with a snapshot build of mongodb-crypt
-//        maven {
-//            url 'https://oss.sonatype.org/content/repositories/snapshots'
-//            content {
-//                includeGroup "org.mongodb"
-//            }
-//        }
+        maven {
```

Comment:
This is a temp change. Will revert once libmongocrypt 1.7.0 binding have been pushed.